apps/src/code-studio/initApp/loadApp.js

```diff
@@ -337,13 +337,13 @@ function loadAppAsync(appOptions) {
       // the header progress data even if the last attempt data takes too long.
       // The progress dots can fade in at any time without impacting the user.
       setTimeout(loadLastAttemptFromSessionStorage, LAST_ATTEMPT_TIMEOUT);
-    } else if (window.dashboard && dashboard.project) {
-      dashboard.project.load().then(function () {
-        if (dashboard.project.hideBecauseAbusive()) {
+    } else if (window.dashboard && project) {
```

Comment:
You can probably just check `window.dashboard` since `project` is explicitly imported and will definitely be truthy.